snp_motion_planning/src/taskflow_generators.hpp

```diff
@@ -4,123 +4,131 @@
 #include <tesseract_motion_planners/simple/simple_motion_planner.h>
 #include <tesseract_motion_planners/ompl/ompl_motion_planner.h>
 // Task Generators
-#include <tesseract_process_managers/task_generators/check_input_task_generator.h>
-#include <tesseract_process_managers/task_generators/has_seed_task_generator.h>
-#include <tesseract_process_managers/task_generators/seed_min_length_task_generator.h>
-#include <tesseract_process_managers/task_generators/motion_planner_task_generator.h>
-#include <tesseract_process_managers/task_generators/discrete_contact_check_task_generator.h>
-#include <tesseract_process_managers/task_generators/iterative_spline_parameterization_task_generator.h>
+//#include <tesseract_process_managers/task_generators/check_input_task_generator.h>
+//#include <tesseract_process_managers/task_generators/has_seed_task_generator.h>
+//#include <tesseract_process_managers/task_generators/seed_min_length_task_generator.h>
+//#include <tesseract_process_managers/task_generators/motion_planner_task_generator.h>
+//#include <tesseract_process_managers/task_generators/discrete_contact_check_task_generator.h>
+//#include <tesseract_process_managers/task_generators/iterative_spline_parameterization_task_generator.h>
```

Comment:
If we can get the planning to work without these custom taskflows, then let's just get rid of this file completely. Otherwise, let's port these to the latest version before merging

Reply:
It works without the custom taskflows, but it would be nice to have custom taskflows for the sake of example. It just became a lot more difficult to make a custom pipeline if you only want to update the raster and freespace planning.

Reply:
Currently have motion planning fully functional with a custom pipeline on 0.14.0. I hid most of the messiness of it in a new directory `snp_motion_planning/src/raster_task`